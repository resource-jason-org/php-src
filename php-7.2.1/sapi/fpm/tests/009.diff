001+ sh: 1: -F: not found
002+ 
003+ Warning: fsockopen(): unable to connect to unix:///home/php-src/php-7.2.1/sapi/fpm/tests/php-fpm.sock:-1 (No such file or directory) in /home/php-src/php-7.2.1/sapi/fpm/tests/fcgi.inc on line 255
004+ UDS error
001- [%d-%s-%d %d:%d:%d] NOTICE: fpm is running, pid %d
002- [%d-%s-%d %d:%d:%d] NOTICE: ready to handle connections
003- int(%d)
004- UDS ok