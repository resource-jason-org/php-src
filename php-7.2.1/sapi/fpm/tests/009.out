sh: 1: -F: not found

Warning: fsockopen(): unable to connect to unix:///home/php-src/php-7.2.1/sapi/fpm/tests/php-fpm.sock:-1 (No such file or directory) in /home/php-src/php-7.2.1/sapi/fpm/tests/fcgi.inc on line 255
UDS error